tests/test_datacard.py

```diff
@@ -0,0 +1,38 @@
+import audbcards
+import pytest
+import difflib
+import numpy as np
+
+from audbcards.core.dataset import create_datasets_page
+
+
+def test_datacard_from_template_lines_similar(db, default_template):
+    """Create datacard using jinja2 via Dataset and Datacard."""
+
+    dataset = audbcards.Dataset(pytest.NAME, pytest.VERSION)
+    dc = audbcards.Datacard(dataset)
+    content = dc._render_template()
+
+    ratios = []
+    for line1, line2 in zip(default_template.split("\n"), content.split("\n")):
+        sm = difflib.SequenceMatcher(lambda x: x == " ", line1, line2)
+        ratios.append(sm.ratio())
+
+    # there are lines that will differ (author, date of dataset)
```

Comment:
Maybe it would be easier to understand and interpret the test result by just removing those lines from both files/strings before comparing them?

Reply:
I am not sure whether I understand: if I remove differing lines from the diff then the result will be identical by definition. 

So you think about removing certain lines a priori and then assert strict identity?

Reply:
My idea was to parse the two string for lines starting with `published`, `repository` and remove those lines before comparing.

Reply:
If this is too complicated I'm also fine with your current solution.

Reply:
This is quite hard.

I used this

```python
    regexes = [r"\npublished[\ \s\w-]+\n",
               r"\nCreated[\ \s\w,]+\n",
               r"\nrepository[=</#>\ \s\w,`-]+"
               ]
    nrepl = 1
    len_c = [len(content)]
    len_t = [len(default_template)]
    for regex in regexes:
        content = re.sub(regex, '', content, nrepl)
        default_template = re.sub(regex, '', default_template, nrepl)
        len_c.append(len(content))
        len_t.append(len(default_template))
```

The number of differences (between len_c end len_t) in characters decreases to 5 but there is still a diff consisting of `\n's that I cannot explain. 


```
['  .. _db:',
 '  ',
 '  db',
 '  --',
 '  ============= ======================',
 '  version       `1.0.0 '
 '<https://github.com/audeering/db/blob/main/CHANGELOG.md>`__',
 '  license       `CC0-1.0 '
 '<https://creativecommons.org/publicdomain/zero/1.0/>`__',
 '  source        https://github.com/audeering/audbcards',
 '  usage         unrestricted',
 '- languages     eng, deu ',
 '?                       -\n',
 '+ languages     eng, deu',
 '  format        wav',
 '  channel       1',
 '  sampling rate 8000',
 '  bit depth     16',
 '  duration      0 days 00:00:04',
 '  files         2^^^^^^^^^^^',
 '  ',
 '  Example database.',
 '  ',
 '  Example',
 '  ^^^^^^^',
 '  ',
 '  :file:`data/f1.wav`',
 '  ',
 '  .. image:: ../db.png',
 '  ',
 '  .. raw:: html',
 '  ',
 '      <p><audio controls src="db/data/f1.wav"></audio></p>',
 '  ',
 '  Tables',
 '  ^^^^^^',
 '  ',
 '  .. csv-table::',
 '     :header: ID,Type,Columns',
 '     :widths: 20, 10, 70',
 '  ',
 '-     "files", "filewise",  "speaker"',
 '?                          -\n',
 '+     "files", "filewise", "speaker"',
 '-     "segments", "segmented",  "emotion"',
 '?                              -\n',
 '+     "segments", "segmented", "emotion"',
 '-     "speaker", "misc",  "age, gender"',
 '?                        -\n',
 '+     "speaker", "misc", "age, gender"',
 '  ',
 '  Schemes',
 '  ^^^^^^^',
 '  ',
 '  .. csv-table::',
 '     :header: ID,Dtype,Min,Labels,Mappings',
 '  ',
 '      "age", "int", "", ""',
 '      "emotion", "str", "", "angry, happy, neutral"',
 '      "gender", "str", "", "female, male"',
 '      "speaker", "int", "", "0, 1", "age, gender"']
```

Reply:
I will make a new comment, and mark this as resolved.